src/smoosh/src/smoosh_channel.erl

```diff
@@ -28,6 +28,11 @@
     terminate/2
 ]).
 
+-define(VIEW_CHANNELS,
+    smoosh_utils:split(
+        config:get("smoosh", "view_channels", "upgrade_views,ratio_views,slack_views")
+    )
+).
```

Comment:
Instead of a macro, I think it might look nicer as `smoosh_utils:is_view_channel(ChanName)` call. So we can keep the splitting and config in that module only

Reply:
Ah, I like that idea!